spark8t/literals.py

```diff
@@ -0,0 +1,3 @@
+MANAGED_BY_LABELNAME = "app.kubernetes.io/managed-by"
+PRIMARY_LABELNAME = "app.kubernetes.io/spark8t-primary"
+SPARK8S_LABEL = "spark8t"
```

Comment:
See my [comment](https://github.com/canonical/spark-k8s-toolkit-py/pull/5#discussion_r1200358184) on this on PR [[DPE-1952] Remove snap references #5](https://github.com/canonical/spark-k8s-toolkit-py/pull/5)

Reply:
Also: instead of hardcoding the name of the package here, I think we should get it in a programmatic way.
(Except if using 'spark8t' is just a "coincidence" here, but not necessarily the name of the package -- in which case this may be good to indicate in a comment, before a next developer like myself may think that module name shouldn't be hardcoded as a literal inside the package :-) )

Reply:
These are constants, that should be centralized in a single place. Also, the name `spark8t` of label values may change (as long as it is constant in the package and should not be constraint as the name of the pckage)